docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/TracerV.rst

```diff
@@ -153,7 +154,7 @@ Instruction value trigger
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Trace recording begins when a specific instruction is seen in the instruction
-trace and recording ends when a specific instruction is seen in the instruction
+trace and  when a specific instruction is seen in the instruction
```

Comment:
Any specific reason "recording ends" was deleted here?

Reply:
i botched the undo of my edits. I'll put it back.